### PR TITLE
Ba console access

### PR DESCRIPTION
**Pull request by bernardanderson**

### CODE REVIEW

File: Mathmagician.Tests/UserInterfaceTests.cs [hunk 1]
```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Mathmagician.Tests
+{
+    [TestClass]
+    public class UserInterfaceTests
```
Comment (katerebekah) [inline Mathmagician.Tests/UserInterfaceTests.cs:8]: I see the UserInterface tests here. Ignore what I said in the other PR!!

File: Mathmagician.Tests/UserInterfaceTests.cs [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        {
+            //arrange
+            UserInterface myUserInterface = new UserInterface();
+            myUserInterface.returnedOperationResult = new List<int> { 2, 66, 78 };
+
+            //assert
+            Assert.AreEqual("String", myUserInterface.ReturnedOperationValues().GetType().Name);
+            Assert.AreEqual("2 66 78 ", myUserInterface.ReturnedOperationValues());
+        }
+    }
```
Comment (katerebekah) [inline Mathmagician.Tests/UserInterfaceTests.cs:158]: You can add a couple unit tests that use CollectionAssert.AreEqual to determine if all the items in the lists you are comparing are equal.

**Comment by bernardanderson** [comment]
@katerebekah 
I added two additional CollectionAssert tests using AreEqual.  I also threw in some AreNotEqual's to make sure that they weren't giving false positives.

**Comment by katerebekah** [comment]
:+1:

**Comment by VikingPaul** [comment]
only thing that i see is you have primes and integers, but the rest are singular. but that's not a big deal. in the program.cs you should have something like a ReadLine at the end so we can see the output. otherwise it's good.

**Comment by VikingPaul** [comment]
:+1:

**Pull request merged**